packages/page-contracts/src/index.tsx

```diff
@@ -74,69 +84,62 @@ function ContractsApp ({ basePath, className = '' }: Props): React.ReactElement<
   );
 
   return (
-    <main className={`contracts--App ${className}`}>
-      <HelpOverlay md={introMd as string} />
-      <header>
-        <Tabs
-          basePath={basePath}
-          items={itemsRef.current}
+    <React.Fragment>
```

Comment:
Always prefer a simple `<>` when using Fragment. 

Having said that, for all these you would generally want a `<div className={className}>` everywhere since some do have styles and consistency needs to be applied.

Reply:
Can you provide some use cases or examples of `className` usage on app level? Couldn't find it.

As we want to render `main` in the `apps/src/Content` now, I thought we could possibly just add `<main className={`App--${name}`}>` there.